src/components/data-processing/CellSizeDistribution/CellSizeDistribution.jsx

```diff
@@ -93,8 +93,7 @@ const CellSizeDistribution = (props) => {
     (state) => state.componentConfig[plots[selectedPlot].plotUuid]?.config,
   );
   const expConfig = useSelector(
-    (state) => state.experimentSettings.processing[filterName][sampleId]?.filterSettings
-      || state.experimentSettings.processing[filterName].filterSettings,
+    (state) => state.experimentSettings.processing[filterName][sampleId].filterSettings,
```

Comment:
What if it's not multisample, would we still have this path?

Reply:
this `||` was added because for some of the steps, sample's filter settings weren't defined in the beginning and we only had the default values (stored in `state.experimentSettings.processing[filterName][sampleId].filterSettings`), not because of unisample.

Still using these default values is no longer useful and makes code unnecessarily complex.

In unisample we still use the sample's id, for example you can check and see that (even before this PR) we never modified the default settings, not even for unisample